SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -1,236 +1,71 @@
-%global virtiofsd_binary        virtiofsd
-
-%global runtime_make_vars       DEFMEMSZ=256 \\\
-                                DEFSANDBOXCGROUPONLY=true \\\
-                                DEFSTATICSANDBOXWORKLOADMEM=1792 \\\
-                                DEFSNPGUEST=true \\\
-                                DEFVIRTIOFSDAEMON=%{_libexecdir}/"%{virtiofsd_binary}" \\\
-                                SKIP_GO_VERSION_CHECK=1
-
-%global agent_make_vars         LIBC=gnu \\\
-                                AGENT_POLICY=yes
-
 %global debug_package %{nil}
+%define sourceName kata-containers
 
 Name:         kata-containers-cc
-Version:      3.2.0.azl2
-Release:      7%{?dist}
+Version:      3.2.0.azl3
+Release:      1%{?dist}
 Summary:      Kata Confidential Containers package developed for Confidential Containers on AKS
 License:      ASL 2.0
 URL:          https://github.com/microsoft/kata-containers
 Vendor:       Microsoft Corporation
 Distribution: Azure Linux
-Source0:      https://github.com/microsoft/kata-containers/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:      %{name}-%{version}-cargo.tar.gz
-Source2:      mariner-coco-build-uvm.sh
+Source0:      https://github.com/microsoft/kata-containers/archive/refs/tags/%{version}.tar.gz#/%{sourceName}-%{version}.tar.gz
+Source1:      %{sourceName}-%{version}-cargo.tar.gz
 
 ExclusiveArch: x86_64
 
 BuildRequires:  golang
-BuildRequires:  make
 BuildRequires:  protobuf-compiler
-BuildRequires:  dracut
-BuildRequires:  cargo
 BuildRequires:  rust
-BuildRequires:  git
-BuildRequires:  sudo
-BuildRequires:  perl-FindBin
-BuildRequires:  perl-lib
 BuildRequires:  libseccomp-devel
 BuildRequires:  openssl-devel
 BuildRequires:  clang
 BuildRequires:  device-mapper-devel
 BuildRequires:  cmake
 BuildRequires:  fuse-devel
 
-# needed to build the tarfs module, see next comment - we currently build the tarfs module for both kernels
-BuildRequires:  kernel-uvm-devel
-
-# kernel-uvm is required for allowing to test the kata-cc handler w/o SEV SNP but with the
-# policy feature using kernel-uvm and the kata-cc shim/agent from this package with policy and snapshotter features
+# kernel-uvm is used for testing in a debug configuration w/o SEV SNP enablement
 Requires:  kernel-uvm
```

Comment:
Not required for this PR, but since `kernel-uvm` is only used for testing, should this be separated out into a subpackage?

Reply:
Per discussion, clarified comment.